Exercises-4-OOP/src/inheritanceanimal/Animal.java

```diff
@@ -0,0 +1,28 @@
+package inheritanceanimal;
+
+public class Animal {
+
+	 String name;
+
+	public Animal() {
+
+	}
+
+	public Animal(String name) {
+		this.name = name;
+	}
+
+	public String getName() {
```

Comment:
This method is not part of the requirements, should be removed.

Reply:
the cat class and dog class required this method to run, if I remove it, they can't run.